mitodl/mitxonline
# Adds some additional options and docs for some management commadns

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

Adds a couple things to the management commands:
- `create_courseware` can accept dates 
- `create_discount_code` can now create one-time or one time per user discounts
- Added some documentation for `create_discount_code`

#### How should this be manually tested?

* Attempt to create a course run using `create_courseware` while specifying dates. It should work as long as you provide an actual date. You'll get an exception if you don't provide a valid date.
* Attempt to create a discount code with the one-time or once-per-user flag set. These should be created as expected. 
* The documentation for `create_discount_code` should not have errors.
